pkg/store/store.go

```diff
@@ -42,18 +47,20 @@ const (
 	cacheExpirationMinutes = 30
 )
 
-func NewStore() (*Store, error) {
+func NewStore() error {
 	//nolint:exhaustruct
 	storeMassaStation := &Store{}
```

Comment:
you can assign StoreInstance directly here. if there is an error, the app will crash anyway